packages/core/src/components/MonthPicker/MonthPicker.jsx

```diff
@@ -167,6 +191,7 @@ export class MonthPicker extends React.PureComponent {
 
   render() {
     const { selectAllText, clearAllText } = this.props;
+    const { selectAllPressed, clearAllPressed } = this.state;
```

Comment:
I just realized that this will only set the aria labels if it's an uncontrolled component, but it can also be a controlled component. In the case of it being a controlled component, we need to derive these two booleans like you do in `handleAriaPressed`. Something like
```
if it's a controlled component:
  selectAllPressed = selectedMonths.length === 12
  clearAllPressed = selectedMonths.length === 0
else:
  get them from the state
```

Reply:
And in fact we use it as a controlled component in taxtools

Reply:
@pwolfert Thank you for the code snippet. Updates pushed and ready for review.

Reply:
Perfect 👍